src/generic_core/uproxy_core.ts

```diff
@@ -649,3 +655,17 @@ export class uProxyCore implements uproxy_core_api.CoreApi {
     });
   }
 }  // class uProxyCore
+
+// Invoke an async function, and retry on error, calling func up to
+// maxAttempts number of times.
+export var retry = <T>(func :() => Promise<T>, maxAttempts :number)
+    : Promise<T> => {
```

Comment:
Done, although it's over 80 characters, we ignore that rule elsewhere in this file..